src/flows/pipes/Notification/Measurement.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {useState, FC, useCallback, useContext} from 'react'
+import {
+  ComponentSize,
+  FlexBox,
+  AlignItems,
+  TextBlock,
+  Dropdown,
+  FlexDirection,
+  ComponentStatus,
+} from '@influxdata/clockface'
+
+import {PipeContext} from 'src/flows/context/pipe'
+import 'src/flows/pipes/Notification/Threshold.scss'
+
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+
+interface Props {
+  readOnly?: boolean
+}
+
+const Measurement: FC<Props> = ({readOnly}) => {
+  const {data, update, results} = useContext(PipeContext)
+  const [measurement, setMeasurement] = useState(data.measurement || null)
```

Comment:
please dont mirror state within the view layer like this as it causes weird bugs. just reference `data.measurement` and update the state with the `update` function

Reply:
done.